connector/src/main/java/org/jboss/as/connector/subsystems/datasources/DataSourceModelNodeUtil.java

```diff
@@ -181,13 +180,14 @@ static ModifiableDataSource from(final OperationContext operationContext, final
 
 
         final String securityDomain = ModelNodeUtil.getResolvedStringIfSetOrGetDefault(operationContext, dataSourceNode, SECURITY_DOMAIN);
-        final boolean elytronEnabled = ModelNodeUtil.getBooleanIfSetOrGetDefault(operationContext, dataSourceNode, ELYTRON_ENABLED);
+        if (securityDomain != null) {
+            ConnectorLogger.DS_DEPLOYER_LOGGER.legacySecurityNotSupported();
```

Comment:
This method produces an OperationFailedException but it is not thrown. Here an OFE is ok.

Reply:
fixed